resources/metadata/mavenBuild.yaml

```
@@ -74,6 +74,17 @@ spec:
         default: false
         aliases:
           - name: maven/logSuccessfulMavenTransfers
+      - name: createBOM
+        type: bool
+        description: Creates the bill of materials (BOM) using CycloneDX Maven plugin. This is set to `false` by default.
```

No need to specify the default as it's fetched from `default` property for docs generation.
```suggestion
        description: Creates the bill of materials (BOM) using CycloneDX Maven plugin.
```